### PR TITLE
Added concrete example document

### PR DESCRIPTION
**Pull request by michellesanver**
In a meeting that took place the 21st of February 2018 we defined this list as a start to a concrete example document, to specifically list out some things that are unwanted behaviour.

### CODE REVIEW

**Review by GwendolenLynch — APPROVED** [review]

File: contributing/code_of_conduct_concrete_example_document.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Code of Conduct: Concrete Example Document
+==========================================
+This is a living document that serves to give concrete examples to unwanted behaviour. These examples have all taken place somewhere in the PHP community in the past, and are clear code of conduct violations according to the Symfony code of conduct.
```
Comment (sstok) [inline contributing/code_of_conduct_concrete_example_document.rst]: Documentation lines are usually limited to 72 characters or somewhere near 👍 and headers are followed by a single empty line (sphinxdoc can be picky about this 😄 )

http://symfony.com/doc/current/contributing/documentation/standards.html

File: contributing/code_of_conduct_concrete_example_document.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Concrete Examples
+-----------------
+- Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of ‘dead’ (rejected) names
```
Comment (sstok) [inline contributing/code_of_conduct_concrete_example_document.rst]: Maybe we can add a short explanation about this? Like the one in the referenced source.

**Review by sstok — CHANGES_REQUESTED** [review]
Some minor syntax issues. Content looks good 👍

File: contributing/code_of_conduct_concrete_example_document.rst (hunk 2 above)
Comment (javiereguiluz) [inline contributing/code_of_conduct_concrete_example_document.rst]: The `deadname` concept is not that well-known, so what do you think about linking it to https://en.wiktionary.org/wiki/deadname or adding a brief description like this one?

```rst
- Deliberate misgendering or use of deadnames (which are the original names given to
  transgender people before initiating their transition).
```

**Review by javiereguiluz — APPROVED** [review]
It's really sad to read that all this happened for real 😞. Thanks for contributing it!

File: contributing/code_of_conduct_concrete_example_document.rst (hunk 2 above)
Comment (michellesanver) [reply contributing/code_of_conduct_concrete_example_document.rst]: @sstok Which explanation are you referring to? I'm afraid I can't find it...

File: contributing/code_of_conduct_concrete_example_document.rst (hunk 1 above)
Comment (michellesanver) [reply contributing/code_of_conduct_concrete_example_document.rst]: Changed :)

File: contributing/code_of_conduct_concrete_example_document.rst (hunk 2 above)
Comment (michellesanver) [reply contributing/code_of_conduct_concrete_example_document.rst]: Changed :)

File: contributing/code_of_conduct_concrete_example_document.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Code of Conduct: Concrete Example Document
+==========================================
+
+This is a living document that serves to give concrete examples to
```
Comment (greg0ire) [inline contributing/code_of_conduct_concrete_example_document.rst]: to => of ?

Comment (Pierstoval) [reply contributing/code_of_conduct_concrete_example_document.rst]: Not sure "of" is right in this case

**Review by Pierstoval — APPROVED** [review]

File: contributing/code_of_conduct_concrete_example_document.rst (hunk 3 above)
Comment (GawainLynch) [reply contributing/code_of_conduct_concrete_example_document.rst]: I would say "of" is the correct preposition in this case.

Comment (michellesanver) [reply contributing/code_of_conduct_concrete_example_document.rst]: Yeah, of is better. Changed it. Thanks! :)

**Review by greg0ire — APPROVED** [review]

**Review by egircys — COMMENTED** [review]
Can it be placed under `contributing/code_of_conduct/concrete_example_document.rst`?

**Review by sstok — APPROVED** [review]

**Comment by michellesanver** [comment]
@egircys Of course! Didn't know quite where to place it. I moved it there now 👍

**Comment by greg0ire** [comment]
@michellesanver Travis says you should add that document to the toctree (it's unreachable otherwise).

**Comment by lsmith77** [comment]
realtes to https://github.com/symfony/diversity/issues/2

File: contributing/code_of_conduct/concrete_example_document.rst [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  of a person who has since changed their name, often a transgender person).
+- Threats of violence like "The person that created this PR should be
+  punched in the face"
+- Incitement of violence towards any individual, including encouraging a
+  person to commit suicide or to engage in self-harm
+- Sustained disruption of discussion
+- Pattern of inappropriate social contact, such as requesting/assuming
+  inappropriate levels of intimacy with others
+- Continued one-on-one communication after requests to cease
+- Putting down people based on their technology choices or their work
```
Comment (wouterj) [inline contributing/code_of_conduct/concrete_example_document.rst]: Can you fix some minor syntatic things here:

 * Please use `*` as list indicator
 * Please end all entries with a semi-colon, except from the last item which should end with a final stop (`.`).

File: contributing/code_of_conduct/concrete_example_document.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+-----------------
+
+- Unwelcome comments regarding a person’s lifestyle choices and practices,
+  including those related to food, health, parenting, drugs, and employment.
+- Deliberate misgendering or use of `dead names`_ (The birth name
+  of a person who has since changed their name, often a transgender person).
+- Threats of violence like "The person that created this PR should be
+  punched in the face"
+- Incitement of violence towards any individual, including encouraging a
+  person to commit suicide or to engage in self-harm
```
Comment (fabpot) [inline contributing/code_of_conduct/concrete_example_document.rst]: Would it makes sense to add something like "even as a joke" (or something similar)?

**Review by fabpot — COMMENTED** [review]
Looks like a great list to me. Thanks.

**Review by dunglas — APPROVED** [review]

File: contributing/code_of_conduct/concrete_example_document.rst (hunk 5 above)
Comment (michellesanver) [reply contributing/code_of_conduct/concrete_example_document.rst]: That does make sense, thanks! Added.

File: contributing/code_of_conduct/concrete_example_document.rst (hunk 4 above)
Comment (michellesanver) [reply contributing/code_of_conduct/concrete_example_document.rst]: Thanks! Fixed :)

**Review by egircys — APPROVED** [review]

**Review by lsmith77 — APPROVED** [review]

**Review by lsmith77 — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you very much @michellesanver for taking the time to create the document and everybody for the constructive feedback.

**Pull request merged**